mgmt/src/processor/proc.rs

```diff
@@ -327,28 +377,38 @@ async fn start_grpc_server_unix(
     }
 
     // Create the UNIX socket listener
-    let uds = match UnixListener::bind(socket_path) {
-        Ok(listener) => listener,
+    let listener = match UnixListener::bind(socket_path) {
+        Ok(listener) => {
+            debug!("Bound unix sock to {}", socket_path.display());
+            listener
+        }
         Err(e) => {
             error!("Failed to bind UNIX socket: {}", e);
             return Err(e);
         }
     };
 
     // Set socket permissions if needed
-    match std::fs::set_permissions(socket_path, std::fs::Permissions::from_mode(0o660)) {
-        Ok(_) => debug!("Socket permissions set to 0660"),
+    match std::fs::set_permissions(socket_path, std::fs::Permissions::from_mode(0o666)) {
```

Comment:
Why does it need to be 0666 and not 0660?

Reply:
For testing, user may belong to "other". I don't think this poses  a security threat atm. We can revisit later...